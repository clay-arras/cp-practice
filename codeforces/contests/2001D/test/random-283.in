1
10
12 6 11 17 9 5 13 5 15 10
