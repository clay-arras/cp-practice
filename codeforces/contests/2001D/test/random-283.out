9
12 6 11 17 9 5 13 15 10
